README.md

```diff
@@ -169,13 +174,13 @@ illuminatio 1.1 is confirmed to be working properly with the following kubernete
 - Google Kubernetes Engine, v1.12.8-gke.10
 - kubeadm 1.15.0-00, kubernetes v1.15.2
 
-
 ## References
 
 The logo was created by Pia Blum.
 
 - Example Network Policies are inspired by:
 [kubernetes-network-policy-recipes](https://github.com/ahmetb/kubernetes-network-policy-recipes)
+
```

Comment:
This makes the result look somehow weird. We could not use a list at all here, that looks better IMO, but I ca agree to disagree and leave it as is.

Reply:
PTAL